### DISCUSSION TITLE
VegaFusion ported to Pixi

### DISCUSSION THREAD

**Discussion by jonmmease** [prefix-dev/pixi#300] [Show and tell]
Just wanted to pass along that I've ported my [VegaFusion](https://github.com/hex-inc/vegafusion) project over to using Pixi for development environment management and automation. This project has a Rust core with wrappers in Python, wasm/JavaScript, and Java and I was able to pull in just about everything needed to build and test the core library and all of the language wrappers using pixi.

This dramatically simplifies the development setup instructions and significantly lower the barrier to entry for contributors. Thanks for making this possible!

At some point I'm planning to write something up on the new setup to try to encourage potential contributors to get involved. Let me know if there's anything you'd like to work together on in terms of writing a blog post or adding VegaFusion as an example project.

**Comment by baszalmstra** [comment]
This is fantastic news! 🥳 🎈 💯 

I would love it if you could open a PR against the [Community.md file](https://github.com/prefix-dev/pixi/blob/main/docs/Community.md) to add your project there!

Thanks so much for your involvement already! If you run into any more issues or missing features don't hesitate to contact us!

**Reply by jonmmease** [reply]
Done: https://github.com/prefix-dev/pixi/pull/302
